src/v/kafka/protocol/topic_properties.h

```diff
@@ -23,4 +23,24 @@ inline constexpr std::string_view topic_property_retention_bytes
 inline constexpr std::string_view topic_property_retention_duration
   = "retention.ms";
 
+inline constexpr std::string_view topic_property_cleanup_policy
+  = "cleanup.policy";
```

Comment:
move names moved. should have squashed. thought i submitted the previous commit in a previous pr.